helm_deploy/laa-court-data-adaptor/public-values/test.yaml

```diff
@@ -22,8 +22,8 @@ sqs_region: eu-west-2
 sqs_messaging_enabled: true
 maat_api:
   validation_enabled: true
-  oauth_url: https://maat-cd-api-tst.auth.eu-west-2.amazoncognito.com
-  api_url: https://maat-cd-api.tst.legalservices.gov.uk
+  oauth_url: https://maat-api-test.auth.eu-west-2.amazoncognito.com
```

Comment:
It's useful to have our `stage` environment up to date even though no VCD is connecting to it. Can we copy these changes across to the stage yaml file as well?